client/submitAssignment.js

```diff
@@ -84,7 +100,11 @@ function buildAssignment() {
     }
 }
 
-// Runs the assignment with the given command.
+/**
+ * Validates the command to run and sends POST request to server
+ * to execute or run the uploaded assignment. It logs and displays the response
```

Comment:
we don't need to mention what the server does here. The comments should just be related to what this function does.

Reply:
Okay